MLJFurnitureStore-master/.editorconfig

```diff
@@ -0,0 +1,4 @@
+﻿[*.{cs,vb}]
+
```

Comment:
There are few concerns:
1- the search feature throws error when I search
![Screenshot 2024-04-05 at 1 54 22 PM](https://github.com/lk00044/MLJFurnitureStore/assets/112569246/4f35c1d0-be18-4ccc-a3ed-7914bc3f2212)

2- we shouldn't be searching by Rental ID, as the instruction states "search furniture by furniture id, category, style"

3- There are 64 file changes. That means your changes are altering all those files which shouldn't be the case. To resolve this, I suggest you checkout from master (after you pull latest) then just copy/paste your search furniture files only. Basically, we should only see the search furniture changes

Reply:
Addressed @lk00044 @jonathanuwg